examples/cfdp-cli-udp/common.py

```diff
@@ -76,56 +82,56 @@
 
 
 class CfdpFaultHandler(DefaultFaultHandlerBase):
-    def __init__(self, base_str: str):
+    def __init__(self, base_str: str) -> None:
```

Comment:
is this a new python best practice? seems like a lot of boilerplate

Reply:
I think it's up to the users in this case but I agree that is unnecessary. I have removed the return types from ``__init__``.